marconi/src/Marconi/Indexers.hs

```diff
@@ -87,21 +84,28 @@ initialCoordinator indexerCount =
               <*> newQSemN 0
               <*> pure indexerCount
 
-type Worker = Coordinator -> TChan (ChainSyncEvent (BlockInMode CardanoMode)) -> FilePath -> IO ()
+-- Currently this `a` should stand for [ChainPoint], but I think eventually it should
```

Comment:
What `a` are you referring to?

Reply:
I actually refactored this into what I said in that comment that it should be.